### PR TITLE
apply some cs/jshint.

### PR DESCRIPTION
**Pull request by aitboudad**

### CODE REVIEW

File: src/javascripts/ng-admin/Crud/routing.js [hunk 1]
```diff
@@ -111,7 +115,7 @@ function routing($stateProvider) {
                                 ).map(entry => dataStore.addEntry(references[name].targetEntity().uniqueId + '_values', entry));
                             }
                         }],
-                        entries: ['dataStore', 'view', 'response', 'referenceEntries', function (dataStore, view, response, referenceEntries) {
+                        entries: ['dataStore', 'view', 'response', function (dataStore, view, response) {
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/routing.js]: This one is on purpose, it forces the ordering of the function execution in the `resolve()` logic. Please revert this change.

File: src/javascripts/ng-admin/Crud/routing.js [hunk 2]
```diff
@@ -189,10 +193,10 @@ function routing($stateProvider) {
                             referencedLists[name].targetFields(),
                             referencedLists[name].targetEntity().name(),
                             referencedLists[name].targetEntity().identifier().name()
-                        ).map(entry => dataStore.addEntry(referencedLists[name].targetEntity().uniqueId + '_list', entry))
+                        ).map(entry => dataStore.addEntry(referencedLists[name].targetEntity().uniqueId + '_list', entry));
                     }
                 }],
-                entryWithReferences: ['dataStore', 'view', 'entry', 'referenceEntries', function(dataStore, view, entry, referenceEntries) {
+                entryWithReferences: ['dataStore', 'view', 'entry', function(dataStore, view, entry) {
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/routing.js]: same as above

File: src/javascripts/ng-admin/Crud/routing.js [hunk 3]
```diff
@@ -333,10 +339,10 @@ function routing($stateProvider) {
                             referencedLists[name].targetFields(),
                             referencedLists[name].targetEntity().name(),
                             referencedLists[name].targetEntity().identifier().name()
-                        ).map(entry => dataStore.addEntry(referencedLists[name].targetEntity().uniqueId + '_list', entry))
+                        ).map(entry => dataStore.addEntry(referencedLists[name].targetEntity().uniqueId + '_list', entry));
                     }
                 }],
-                entryWithReferences: ['dataStore', 'view', 'entry', 'referenceEntries', function(dataStore, view, entry, referenceEntries) {
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/routing.js]: same as above

**Comment by fzaninotto** [comment]
Nice cleanup, looking forward to merging this one.

File: src/javascripts/ng-admin/Crud/routing.js (hunk 1 above)
Comment (aitboudad) [inline src/javascripts/ng-admin/Crud/routing.js]: reverted

**Comment by fzaninotto** [comment]
Thanks!
